uitests/tests/org.jboss.tools.fuse.ui.bot.tests/src/org/jboss/tools/fuse/ui/bot/tests/ConfigurationsEditorDataFormatTest.java

```diff
@@ -102,81 +107,53 @@
 	 */
 	@Parameters
 	public static Collection<String> setupData() {
-
-		new WorkbenchShell();
-		ProjectFactory.newProject(PROJECT_NAME).deploymentType(STANDALONE).runtimeType(KARAF)
-				.version(CAMEL_2_17_0_REDHAT_630187).template(CBR_SPRING).create();
-		for (IViewReference viewReference : WorkbenchPartLookup.getInstance().findAllViewReferences()) {
-			if (viewReference.getPartName().equals("Welcome")) {
-				final IViewReference iViewReference = viewReference;
-				Display.syncExec(new Runnable() {
-					@Override
-					public void run() {
-						iViewReference.getPage().hideView(iViewReference);
-					}
-				});
-				break;
-			}
-		}
+		createProject();
 		new CamelProject(PROJECT_NAME).openCamelContext(CONTEXT);
-		List<String> dataFormats = CamelDataFormatDialog.getDataFormats();
-		return dataFormats;
+		CamelEditor.switchTab("Configurations");
+		return ConfigurationsEditor.getDataFormats();
 	}
 
-	/**
-	 * Utilizes passing parameters using the constructor
-	 * 
-	 * @param template
-	 *            a Global element - Data Format
-	 */
-	public ConfigurationsEditorDataFormatsTest(String element) {
-		this.element = element;
+	public ConfigurationsEditorDataFormatTest(String dataFormat) {
+		this.dataFormat = dataFormat;
+		this.dataFormatName = label(dataFormat);
+		path = new String[] { ROOT, this.dataFormatName, " (Data Format)" };
 	}
 
-	/**
-	 * Prepares test environment
-	 */
 	@BeforeClass
 	public static void setupResetCamelContext() {
+		new WorkbenchShell().maximize();
+		new CleanWorkspaceRequirement().fulfill();
```

Comment:
Is it necessary? You use class annotation _CleanWorkspaceRequirement_, therefore it should be fulfilled before tests ;-).